### PR TITLE
[AMDGPU][True16][test] added missing true16 flag in gfx12 asm vop1

### PR DESCRIPTION
**Pull request by broxigarchen**
added missing true16 flag in gfx12 asm vop1

### CODE REVIEW

**Comment by broxigarchen** [comment]
Hi @Sisyph @kosarev This PR is ready for review. Thanks!

**Review by kosarev — APPROVED** [review]
LGTM!

File: llvm/test/MC/AMDGPU/gfx12_asm_vop1.s [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 // RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize64,+real-true16 -show-encoding -comment-column=0 %s | FileCheck --strict-whitespace --check-prefixes=GFX12,GFX12-ASM %s
-// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize64,+real-true16 -show-encoding %s | %extract-encodings | llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize64 -disassemble -show-encoding -comment-column=0 | FileCheck --strict-whitespace --check-prefixes=GFX12,GFX12-DIS %s
+// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize64,+real-true16 -show-encoding %s | %extract-encodings | llvm-mc -triple=amdgcn -mcpu=gfx1200 -mattr=+wavefrontsize64,+real-true16 -disassemble -show-encoding -comment-column=0 | FileCheck --strict-whitespace --check-prefixes=GFX12,GFX12-DIS %s
 
 // this file will be converted to true16 format when more true16 instructions are supported
```
Comment (Sisyph) [inline llvm/test/MC/AMDGPU/gfx12_asm_vop1.s:6]: Can this comment now be removed?

Comment (broxigarchen) [reply llvm/test/MC/AMDGPU/gfx12_asm_vop1.s:6]: Yes. I will remove this in the following PR

**Pull request merged**